Jenkinsfile

```diff
@@ -17,10 +17,27 @@ pipeline {
 
     stages {
         stage('Build Docker Image') {
-            steps {
-                deleteDir()
-                checkout scm
-                sh 'make build'
+            parallel {
+                stage('Windows') {
+                    agent {
+                        label "windock&&windows"
+                    }
+                    steps {
+                        deleteDir()
+                        checkout scm
+                        bat "powershell -File ./make.ps1"
```

Comment:
Does the `powershell` step work?

Reply:
I guess it should since that's part of the workflow-durable-task-step plugin. I'll fix this up